helmfile.d/stacks/monitoring.yaml.gotmpl

```diff
@@ -49,6 +49,8 @@ templates:
       app: metrics-server
     values:
       - values/metrics-server.yaml.gotmpl
+    wait: true
+    timeout: 300
```

Comment:
Why do we need a wait for the metrics server? I thought that it was not really needed for other components to start up properly, but I might very well be wrong about that.
If I understand your PR description correctly this was related to falco waiting for the monitoring stack. Have you seen any scenario where the metrics server not being ready prevented falco from running?